hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystem.java

```diff
@@ -363,6 +366,20 @@ public boolean delete(final Path f, final boolean recursive) throws IOException
 
   }
 
+  @Override
+  public ContentSummary getContentSummary(Path f) throws IOException {
+    org.apache.hadoop.fs.azurebfs.utils.ContentSummary contentSummary = null;
```

Comment:
Do we need to specify the full package name? avoid if possible.

Reply:
renamed to ABFSContentSummary